src/imageLoader/colorSpaceConverters/convertRGBColorByPlane.js

```diff
@@ -7,9 +7,13 @@ export default function (imageFrame, rgbaBuffer) {
   }
 
   const numPixels = imageFrame.length / 3;
+
   let rgbaIndex = 0;
+
   let rIndex = 0;
+
   let gIndex = numPixels;
+
```

Comment:
Why?

Reply:
Ran linting... We should add husky / prettier / eslint to this library to avoid this.